packages/api/prisma/schema.prisma

```diff
@@ -0,0 +1,38 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+  output   = "./client"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Book {
+  id      Int       @id
+  name    String
+  search  String    @unique
```

Comment:
Ok, that sounds great. It's not immediately clear from reading the schema file though, so it would be good to include an explanation comment for that field.